samwise.js

```diff
@@ -2,6 +2,8 @@
  * Autoplay plugin - Automatically advance slideshow after N seconds
```

Comment:
SCRIPT-I'm impressed with your use of design patterns here, it really enhances the code organization.